### PR TITLE
chore: improve mobile signup page

### PR DESCRIPTION
**Pull request by zlwaterfield**
## Problem

Improving the mobile styling for the signup form

| before | after |
| ------- | ------- |
| Screenshot 2024-03-21 at 1 32 59 PM | Screenshot 2024-03-21 at 1 31 10 PM |

### CODE REVIEW

**Comment by zlwaterfield** [comment]
I'm not entirely sure why all the snapshots were regenerated, the images look identical.

File: frontend/__snapshots__/scenes-other-invitesignup--cloud--dark.png
Comment (xrdt) [inline frontend/__snapshots__/scenes-other-invitesignup--cloud--dark.png:1]: Is it expected that the colors are inverted here?

Comment (xrdt) [reply frontend/__snapshots__/scenes-other-invitesignup--cloud--dark.png:1]: It looks like the dark mode snapshots are generally looking a bit strange.

**Review by xrdt — APPROVED** [review]
doesn't have to be you but we should fix the dark mode snapshots at some point

**Comment by zlwaterfield** [comment]
> doesn't have to be you but we should fix the dark mode snapshots at some point

Agree, I'll add it to my list to look into.

**Pull request merged**